dev/publicDev/Complaint/ctrl.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
```

Comment:
follow the same convention for file names (complaintController or complaint-controller)...